### DISCUSSION TITLE
Query for multiday events

### DISCUSSION THREAD

**Discussion by luke-nehemedia** [solspace/craft-calendar#242] [Feature Requests]
Is there any way to query only multiday-events?

Something like:
```
{% set eventOptions = {
	dateRangeStart: startDate,
	dateRangeEnd: endDate,
	multiday: 1,
} %}
```

**Comment by kjmartens** [comment]
Unfortunately there isn't a way to do this currently. We'll definitely note this for future consideration, however. 🙂
